utils/node_local_test.py

```diff
@@ -748,7 +748,10 @@ def start(self):
 
         agent_config = join(self.agent_dir, 'nlt_agent.yaml')
         with open(agent_config, 'w') as fd:
-            agent_data = {'access_points': scyaml['access_points']}
+            agent_data = {
+                'access_points': scyaml['access_points'],
+                'control_log_mask': 'NOTICE',  # INFO logs every client process connection
```

Comment:
Was NLT using INFO level before? Too spammy now?

Reply:
As we're now logging each application connection at INFO level, yes it did become pretty spammy without any benefit IMO. NLT is kind of a special case in that it's doing fault injection and making client connections as fast as it can (many many times per second).

Reply:
Hmm, I do wonder if that's a sign that logging every client connection at INFO may make INFO too noisy. I guess we'll find out.

Reply:
That change came per a comment in slack by Kevin that they want to be able to see which interface was chosen for a given client process.

Reply:
Ah, yes, I remember that discussion. I had added a log entry in the client logs for that as well.